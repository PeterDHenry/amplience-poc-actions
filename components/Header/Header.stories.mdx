import { Meta, Story, Canvas } from '@storybook/addon-docs/blocks'
import Header from './Header'

<Meta title="Layout/Checkout Header" component={Header} />

# Header

The website header

<Canvas>
  <Story name="Default">
    <Header />
  </Story>
</Canvas>
