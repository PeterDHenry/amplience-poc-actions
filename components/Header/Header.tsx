import React from 'react'
import styled from 'styled-components'
import 'bootstrap/dist/css/bootstrap.min.css'

const Logo = styled.img`
  height: 60px;
  padding: 10px;
`

interface Props {
  title?: string
}

export default function Header() {
  return (
    <header className="container-fluid bg-dark">
      <div className="container text-center">
        <div className="col-xs-12">
          <a href="/" data-ga-action="Logo" itemProp="url">
            <Logo
              src="//www.wigglestatic.com/images/ui/wiggle-logo/desktop-wiggle_master_rgb_logo.svg"
              itemProp="logo"
            />
          </a>
        </div>
      </div>
    </header>
  )
}
