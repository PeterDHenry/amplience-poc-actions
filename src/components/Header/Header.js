import React from 'react';

const Header = () => (
  <div className="bg-gray-800 row-auto p-2.5">
    <div className="grid grid-cols-1 md:grid-cols-12 gap-4">
      <div className="col-span-1 text-center mb-6 md:mb-0">
        <a id="wiggle-logo" href="/" itemProp="url" tabIndex="1">
          <img class="bem-header__logo" src="//www.wigglestatic.com/images/ui/wiggle-logo/desktop-wiggle_master_rgb_logo.svg" itemProp="logo" />
        </a>
      </div>
      <div className="col-span-11 text-center md:text-right">
      </div>
    </div>
  </div>
);

export default Header;
