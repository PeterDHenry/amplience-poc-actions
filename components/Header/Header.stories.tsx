import { Meta, Story } from '@storybook/react'
import { Header as Component } from './Header'

const meta: Meta = {
  title: 'Layout/Checkout',
  component: Component,
  argTypes: {},
}

export default meta

export const Header: Story = () => <Component />
