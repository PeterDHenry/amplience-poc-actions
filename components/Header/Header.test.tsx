import { render } from '@testing-library/react'
import { Header } from './Header'

describe('Header component', () => {
  it('matches snapshot', () => {
    const { asFragment } = render(<Header />, {})
    expect(asFragment()).toMatchSnapshot()
  })

  it('should render header element', () => {
    const { container } = render(<Header />)
    expect(container.getElementsByTagName('header')).toBeTruthy()
  })
})
