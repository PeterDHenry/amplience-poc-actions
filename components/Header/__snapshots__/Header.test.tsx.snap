// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component matches snapshot 1`] = `
<DocumentFragment>
  <header
    class="container-fluid bg-dark"
  >
    <div
      class="container text-center"
    >
      <div
        class="col-xs-12"
      >
        <a
          data-ga-action="Logo"
          href="/"
          itemprop="url"
        >
          <img
            class="Header__Logo-sc-11c03b2-0 tmHxh"
            itemprop="logo"
            src="//www.wigglestatic.com/images/ui/wiggle-logo/desktop-wiggle_master_rgb_logo.svg"
          />
        </a>
      </div>
    </div>
  </header>
</DocumentFragment>
`;
